src/industry_cmd.cpp

```diff
@@ -554,171 +554,194 @@ static bool TransportIndustryGoods(TileIndex tile)
 	return moved_cargo;
 }
 
+static void AnimateSugarSieve(TileIndex tile)
+{
+	byte m = GetAnimationFrame(tile) + 1;
```

Comment:
Someone could argue `m` is not a good variable name ;)

Reply:
Yeah, I argued that myself as well. Though I left it out to not make the change more difficult to review.